forge-gui/res/cardsfolder/upcoming/day_of_the_moon.txt

```diff
@@ -0,0 +1,10 @@
+Name:Day of the Moon
+ManaCost:2 R
+Types:Enchantment Saga
+K:Saga:3:DBNameCard,DBNameCard,DBNameCard
+SVar:DBNameCard:DB$ NameCard | Defined$ You | ValidCards$ Card.Creature | ValidDesc$ Creature | SubAbility$ DBRemember | SpellDescription$ Choose a creature card name, then goad all creatures with a name chosen for Day of the Moon. (Until your next turn, they attack each combat if able and attack a player other than you if able.)
+SVar:DBRemember:DB$ Pump | RememberObjects$ ValidAll Card.NamedCard | SubAbility$ DBGoadAll
```

Comment:
this doesn't really work, creatures can change names

Reply:
Hmm I used null chamber a template but I guess that's not possible here. 

Is this scriptable without engine changes ?

Reply:
probably something for @Northmoc

Reply:
Looks neat, I can take a stab at it

Reply:
I'll remove this one from my PR  then